---
 container/archive.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/container/archive.py b/container/archive.py
index 0c875a3..d70696b 100644
--- a/container/archive.py
+++ b/container/archive.py
@@ -145,7 +145,7 @@ class TarFileWriter(object):
       mode=mode,
       fileobj=self.fileobj,
       # https://github.com/bazelbuild/rules_docker/issues/1602
-      format=tarfile.PAX_FORMAT)
+      format=tarfile.USTAR_FORMAT)
     self.members = set([])
     self.directories = set([])
 
-- 
2.27.0

